MovieChecker/MovieChecker/BoxOfficeTableCell.h

```diff
@@ -0,0 +1,16 @@
+//
+//  BoxOfficeTableCell.h
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface BoxOfficeTableCell : UICollectionViewCell
+
+- (void)setThumbnailImageFromURL:(NSURL *)url;
+- (void)setName:(NSString *)name;
```

Comment:
As with the other cell, I'd say it's preferable to have a method like `displayMovie:` that takes a model object.